src/scripts/main.js

```diff
@@ -1,5 +1,39 @@
 'use strict';
 
+const spider = document.querySelector('.spider');
+const spiderX = spider.clientWidth;
+const spiderY = spider.clientHeight;
+
+const wall = document.querySelector('.wall');
+const wallX = wall.clientWidth;
+const wallY = wall.clientHeight;
```

Comment:
```suggestion
const spiderWidth = spider.clientWidth;
const spiderHeight = spider.clientHeight;

const wall = document.querySelector('.wall');
const wallWidth = wall.clientWidth;
const wallHeight = wall.clientHeight;
```
Let's use more meaningful names